e2e/testFunctions.ts

```diff
@@ -7,6 +7,10 @@ import {
 
 /* eslint-disable sonarjs/no-duplicate-string  -- ignoring duplicate strings here */
 
+// Timeout constants
+const EXPORT_REQUEST_TIMEOUT = 60000;
+const DOWNLOAD_TIMEOUT = 10000;
```

Comment:
Super small nit: would you be able to rename these to `TIMEOUT_EXPORT_REQUEST` and `TIMEOUT_DOWNLOAD`?